hpcgap/lib/mapping.gi

```diff
@@ -195,9 +202,15 @@ InstallGlobalFunction( Image, function ( arg )
 
         return [];
 
+      else
+        ErrorNoReturn( "couldn't match families of <elm> or <coll> and ",
```

Comment:
Why "couldn't match families" and not "families of ... don't match"?

Reply:
I'll use "families of .. don't match". Sounds better. :+1: